vkconfig_core/path_manager.cpp

```diff
@@ -306,7 +306,13 @@ QString PathManager::SelectPathImpl(QWidget* parent, Path path, const QString& s
                 return "";
 
             SetPath(path, QFileInfo(selected_path).absolutePath());
+#if PLATFORM_WINDOWS
             return GetFullPath(path, QFileInfo(selected_path).baseName());
+#else
+            // On Linux, we have no extension to trim off. On macOS, we may or may not have a
+            // .app extension. If the later, it must be preserved at this point.
+            return GetFullPath(path, QFileInfo(selected_path).absoluteFilePath());
```

Comment:
This is a little weird as GetFullPath expect the filename not a path. So in theory, it should be QFileInfo(selected_path).filename() or something like this instead and actually that should work on Windows too.